src/components/FormUseWithValidation/FormUseWithValidation.js

```diff
@@ -0,0 +1,144 @@
+import React, { useCallback } from 'react';
+
+const FormUseWithValidation = (submitFunction) => {

```

Comment:
custom hooks should be stored in folder `hooks` outside `components` because they are not components

Reply:
custom hooks should always start with verb `use` to let `React` know that it's a hook


```js

useFormUseWithValidation


```